packages/common/src/services/dictionaryService.ts

```diff
@@ -0,0 +1,30 @@
+import { isEmpty } from 'lodash-es';
+import { Dependencies } from '../config/config.js';
+
+import getCategoryUtils from '../utils/categoryUtils.js';
+import getDictionaryUtils from '../utils/dictionaryUtils.js';
+
+const service = (dependencies: Dependencies) => {
+	return {
+		registerDictionary: async (categoryName: string, dictionaryName: string, version: string) => {
```

Comment:
inclined to think when this is imported elsewhere it should go as `*.dictionaryService.register` rather than `*.service.registerDictionary`... unless it is meant to be `*.dictionaryService.registerDictionary`, which sounds redundant, so my question: is this service expected to register anything else but dictionaries?

Reply:
now that you mention, it makes more sense to name it '*.dictionaryService.register' as there's nothing else to register.